spring-security-kotlin-dsl/pom.xml

```diff
@@ -17,10 +17,12 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-security</artifactId>
+            <version>${spring-boot.version}</version>
```

Comment:
I think it should be enough to just override the spring-boot.version in the properties section. No need to explicitly set version here. Parent project should handle it correctly. Can you please check it?

Reply:
When I remove explicit declaration, it points to 2.3.3.release 
screenshot attached
![2 3 3 release](https://user-images.githubusercontent.com/4970699/208480028-b9f50056-c6a1-458e-a15d-244fb26837af.png)

Reply:
Don't look at what IntelliJ shows. It's enough to overwrite the property. Once you rebuild the project the correct versions of modules will be used. See printscreen:
![image](https://user-images.githubusercontent.com/10830473/208897042-bd72f674-5592-4c09-957a-6ab80fa12aff.png)

Reply:
Changes done
Thanks